ui/kotlinx-coroutines-android/test/HandlerDispatcherTest.kt

```diff
@@ -116,9 +116,7 @@ class HandlerDispatcherTest : TestBase() {
         mainLooper.runOneTask()
         expect(4)
         mainLooper.scheduler.advanceBy(50, TimeUnit.MILLISECONDS)
-        expect(5)
-        mainLooper.scheduler.advanceBy(51, TimeUnit.MILLISECONDS)
-        finish(6)
+        finish(5)
```

Comment:
Then it's `finish(4)`, as the action after `expect(4)` also does nothing.

Reply:
Done